torchaudio/datasets/lrs3.py

```diff
@@ -0,0 +1,76 @@
+import os
```

Comment:
@mpc001  can you open a separate PR for this file, with thorough documentation and units tests (refer to other examples in the dataset folder), or keep it in the current PR, move it to the example folder?

Reply:
@mpc001 yes, if you'd like to introduce LRS3 to the torchaudio library, we'd request excluding the file from this PR and opening a separate PR for it. We'd also request some additional changes, e.g. code mods, documentation, tests, since the API will be public and expected to be longer-lived. This might take some time to do though, so if you'd prefer, we can first keep the dataset in this PR but move it to the examples folder, and then in a future PR add the dataset to torchaudio and migrate this recipe to it.